src/geocoder_control.js

```diff
@@ -7,6 +7,7 @@ var GeocoderControl = L.Control.extend({
     includes: L.Mixin.Events,
 
     options: {
+        proximity: false,
```

Comment:
To me that would be a breaking change to the API as one is going to get drastically different results than with it set to false. Would it be better to save this for `3.0.0`?